tests/scripts/thread-cert/command.py

```diff
@@ -57,3 +57,22 @@ def check_link_accept(command_msg, destination_node):
 
     destination_link_local = destination_node.get_ip6_address(config.ADDRESS_TYPE.LINK_LOCAL)
     assert ipv6.ip_address(destination_link_local) == command_msg.ipv6_packet.ipv6_header.destination_address, "Error: The destination is unexpected"
+
+def check_icmp_path(sniffer, path, nodes):
```

Comment:
rename to `check_ping_request_path`? as you are only checking `ICMP_ECHO_REQUEST` in the implementation.

Reply:
I will modify it to be able to check every type icmp message forward path.